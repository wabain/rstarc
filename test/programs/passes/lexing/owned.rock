( Test some more complex cases for owned vs borrowed values )
Union's been on strike
Union 's been on strike
A '''' wakin'up Sleepin' I''n
It ain't nothing
