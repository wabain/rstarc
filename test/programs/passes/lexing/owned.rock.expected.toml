[tests]
enabled = ["tokens"]

[tokens]
returncode = "0"
stdout = """
1    ( Test some more complex cases for owned vs borrowed values )⏎

    61..62 ⏎

2    Union's been on strike⏎

    62..67 "Union" borrowed ProperVar
    67..69 is
    70..84 426.0 NumberLiteral
    84..85 ⏎

3    Union 's been on strike⏎

    85..90 "Union" borrowed ProperVar
    91..93 is
   94..108 426.0 NumberLiteral
  108..109 ⏎

4    A '''' wakin'up Sleepin' I''n⏎

  109..110 "a" owned CommonPrep
  116..124 "wakinup" owned CommonVar
  125..138 "Sleepin In" owned ProperVar
  138..139 ⏎

5    It ain't nothing⏎

  139..141 "It" borrowed Pronoun
  142..147 isn't
  148..155 null
  155..156 ⏎

6    ⏎

  156..156 <eof>
"""
